test/nurse_ussd.test.js

```diff
@@ -257,14 +257,56 @@ describe("app", function() {
                                 '1. Subscribe a friend',
                                 '2. Change your no.',
                                 '3. Change facility code',
-                                '4. Change SANC no.',
-                                '5. Change Persal no.',
-                                '6. Stop SMS'
+                                '4. Change ID no.',
+                                '5. Change SANC no.',
+                                '6. More'
                             ].join('\n')
                         })
                         .run();
                 });
-            });
+                it("should give 4 options when user selects more", function() {
+                    return tester
+                        .setup.user.addr('27821237777')
+                        .setup.char_limit(140)  // limit first state chars
+                        .inputs(
+                            {session_event: 'new'},  // dial in
```

Comment:
Put the comma at the end of this line at the beginnning of the next line. This helps a lot with multiple cursor usage.  (Apply this everywhere)